<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>memo on のんびりと生きたい</title>
    <link>http://cloud.yoneyan.net/categories/memo/</link>
    <description>Recent content in memo on のんびりと生きたい</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>ja</language>
    <lastBuildDate>Sun, 31 Mar 2019 15:17:42 +0900</lastBuildDate>
    
	<atom:link href="http://cloud.yoneyan.net/categories/memo/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>LinuxでRAID構築する際にリビルドが遅すぎる時の対処法</title>
      <link>http://cloud.yoneyan.net/post/what-to-do-when-rebuilding-is-too-slow-when-building-raid-on-linux/</link>
      <pubDate>Sun, 31 Mar 2019 15:17:42 +0900</pubDate>
      
      <guid>http://cloud.yoneyan.net/post/what-to-do-when-rebuilding-is-too-slow-when-building-raid-on-linux/</guid>
      <description>CentOSで1TBのHDDをRAID1に構築する際にかなり時間が掛かりそうだったので、対処法の紹介です。
http://liliumrubellum.blog10.fc2.com/blog-entry-300.html
上のブログさんの記事を参考にしました。（ほぼ、同じです。）
備忘録として、こちらに書かせていただきます。
どれくらいの時間がかかるのか cat /proc/mdstat  と実行すると、RAID状況を確認できます。
改善前 Personalities : [raid1] md0 : active raid1 sdc1[1] sdb1[0] 976628736 blocks super 1.2 [2/2] [UU] [&amp;gt;....................] resync = 2.1% (21226880/976628736) finish=1186.8min speed=13416K/sec bitmap: 8/8 pages [32KB], 65536KB chunk  13M/secはあまりにも遅すぎるので、これをなのとかします。
改善後 Personalities : [raid1] md0 : active raid1 sdc1[1] sdb1[0] 976628736 blocks super 1.2 [2/2] [UU] [&amp;gt;....................] resync = 4.2% (41735552/976628736) finish=292.9min speed=53181K/sec bitmap: 8/8 pages [32KB], 65536KB chunk  改善後はかなり速度が上がりました。</description>
    </item>
    
    <item>
      <title>Edgerouterでポート開放をしてみる</title>
      <link>http://cloud.yoneyan.net/post/try-opening-the-port-with-edgerouter/</link>
      <pubDate>Fri, 22 Mar 2019 18:05:10 +0900</pubDate>
      
      <guid>http://cloud.yoneyan.net/post/try-opening-the-port-with-edgerouter/</guid>
      <description>今回はEdgerouterでポート開放をコマンド操作でしていきます。
こちらのサイトを参考にさせていただきました。
https://qiita.com/maiani/items/b45b0505377401be083a
わかりにくい点などがありましたら、上記のサイトさんをご覧ください。
コマンドで設定していく 今回は以下のポートを開放していきます。
80/tcp 172.16.1.100
3389/tcp,3389/udp 172.16.1.10
1．ファイアウォールの設定 80/tcpの設定をしていきます。
set firewall name WAN_IN rule 50 description &amp;quot;80/tcp http&amp;quot; set firewall name WAN_IN rule 50 action accept set firewall name WAN_IN rule 50 destination address 172.16.1.100 set firewall name WAN_IN rule 50 destination port 80 set firewall name WAN_IN rule 50 protocol tcp  3389/tcpと3389/udpポートを設定します。
set firewall name WAN_IN rule 50 description &amp;quot;3389/tcp,3389/udp RDP&amp;quot; set firewall name WAN_IN rule 50 action accept set firewall name WAN_IN rule 50 destination address 172.</description>
    </item>
    
    <item>
      <title>neovimでUltiSnips requires py &gt;= 2.7 or py3というエラーの対処法</title>
      <link>http://cloud.yoneyan.net/post/how-to-fix-neovim-error/</link>
      <pubDate>Wed, 20 Feb 2019 18:53:54 +0900</pubDate>
      
      <guid>http://cloud.yoneyan.net/post/how-to-fix-neovim-error/</guid>
      <description>最近、新たにデスクトップPCにLinuxMintを導入してneovimやtmuxやらの環境構築をしている途中に
UltiSnips requires py &amp;gt;= 2.7 or py3
とこのようなエラーをneovimを起動する度に吐いてしまったので対処法を備忘録として載せておきます。
環境 OS:LinuxMint 19
原因 apt-get install neovim からインストールしたため、neovimの最新バージョンではないのが原因
コマンドを入力 pip install neovim  これを入力すると、
Collecting neovim Using cached https://files.pythonhosted.org/packages/78/ec/ac9905ccab8774b64c37cdff9e08db320c349eda0ae3161aebcac83e5590/neovim-0.3.1.tar.gz Complete output from command python setup.py egg_info: Traceback (most recent call last): File &amp;quot;&amp;lt;string&amp;gt;&amp;quot;, line 1, in &amp;lt;module&amp;gt; ImportError: No module named setuptools ---------------------------------------- Command &amp;quot;python setup.py egg_info&amp;quot; failed with error code 1 in /tmp/pip-build-bezjAu/neovim/  このような謎エラーが出てきました。
neovimとpip installで入れようとしているものとバージョンが合わないと怒られます。
ということで、neovimの公式からインストールしていきましょう。https://github.com/neovim/neovim/wiki/Installing-Neovim
LinuxMintの場合はベースのディストリビューションがUbuntuになるので、Ubuntuのインストール方法を用います。</description>
    </item>
    
    <item>
      <title>wordpressでGutenbergエディタが開いてくれない</title>
      <link>http://cloud.yoneyan.net/post/gutenberg-editor-does-not-open-in-wordpress/</link>
      <pubDate>Wed, 30 Jan 2019 19:41:05 +0900</pubDate>
      
      <guid>http://cloud.yoneyan.net/post/gutenberg-editor-does-not-open-in-wordpress/</guid>
      <description>最近、Wordpressで標準搭載されたGutenbergエディタという世間では賛否両論になっているものがあります。 個人的には好きなエディタですが、久しぶりに書き溜めていた記事をもう一度書こうと思い開いてみると「エディタで予期しないエラーが発生しました」と出てきてかなり焦りました。
結果として、なんとか問題は解消されましたがそこまで大したことはしていないのですが、Wordpressがバグったときに使えると思うので、備忘録として残します。
このようになった    Gutenbergが開いてくれない
上の画像にもあるように、「エディターで予期しないエラーが発生しました」
「復旧を試みる」をクリックしても何の反応もありませんでした。
「投稿内容をコピー」や「エラー文をコピー」はクリックすると反応がありしっかりとコピーできました。
たまたま、開いている記事だけがおかしいのかと思い、いくつか他の記事を開いても同じような現象が発生しました。
プラグインを疑う 最初にプラグインを疑いました。
すると、まさかのドンピシャでした。
   すべてのプラグインを停止
すべてのプラグインを停止させてから一つずつ有効にさせていくと&amp;hellip;
   プラグイン一覧
Gutenbergプラグインが一番怪しいということがわかりました。
おそらく、標準搭載される前に、このプラグインを使っていたので、これがバージョンが上がった段階でこのような現象が起こったことが今回の原因ではないかと思っています。
結論 プラグインはしっかり管理しておいた方がいいです
私の場合は、プラグインの関連でこの問題が起こりました。逆に、プラグインを全部無効にしてもGutenbergエディタが開けない場合は他の問題が起因している可能性が高いです。</description>
    </item>
    
    <item>
      <title>asteriskでUnable to retrieve PJSIP transport &#39;simpletrans&#39;というエラーが出る</title>
      <link>http://cloud.yoneyan.net/post/asterisk-error1/</link>
      <pubDate>Sun, 04 Nov 2018 03:18:33 +0900</pubDate>
      
      <guid>http://cloud.yoneyan.net/post/asterisk-error1/</guid>
      <description>Unable to retrieve PJSIP transport &amp;lsquo;simpletrans&amp;rsquo;というエラーについて 結果を言うと、だれでもわかるような超単純なミスでした。[simpletrans]というtransportセクションが見つからないという意味です。
解決方法 超簡単です。transportの部分を書き換えてあげるだけでいいです。
[transport-udp] type=transport protocol=udp bind=0.0.0.0 [acl] permit=0.0.0.0/0.0.0.0 [100] type=endpoint transport=simpletrans allow=all aors=100 auth=auth100 [100] type=aor max_contacts=1 [auth100] type=auth auth_type=userpass password=password username=100  transport=simpletransの部分をtransport-udpに書き換えてあげることで問題が解決します。内線番号で指定できる理由は複数のtransportがあった場合、どれがどれを使えばいいかわからなくなるのでここで指定する必要があるらしいです。
まとめ これは誰でもわかるようなミスなのですが、解決するまでかなりの時間を要したので備忘録として残しておきます。</description>
    </item>
    
    <item>
      <title>Windows10 Fall Creators Updateでonedriveのファイルオンデマンドが使えない場合の対処法</title>
      <link>http://cloud.yoneyan.net/post/what-to-do-if-you-cannot-use-onedrive-file-on-demand-with-windows10fallcreatorsupdate/</link>
      <pubDate>Sun, 22 Oct 2017 04:25:45 +0900</pubDate>
      
      <guid>http://cloud.yoneyan.net/post/what-to-do-if-you-cannot-use-onedrive-file-on-demand-with-windows10fallcreatorsupdate/</guid>
      <description>つい最近、年に二回のwindows10鬼の大型アップデートの時期が来てしまいました。 普通はwindows updateを待つべき何ですが、いきなりアップデートされては困るのでいつも手動でアップデートさせている筆者です。年一回で十分のように思ってしまうのは自分だけなのかなｗ 今回は、アップデートしたのはいいんですが、今回のアップデートの醍醐味のひとつであるonedriveのファイルオンデマンドが使えないので、その対処法についてこの記事ではご紹介していきます。  

Windows 10 Fall Creators Updateにアップデートさせたのになんでないねんｗって普通なるよねー
そもそも、なぜこうなってしまうのか？ onedriveのバージョンが古いというのが一番の影響になります。 一番、最新のonedriveに更新する必要があります。 そもそも、なんでアップデートしたときにonedriveの最新版を入れないのかがよくわからないんですがｗ ここではWindows 10 Fall Creators Updateを更新したという前提で手順を説明させていただきます。
手順 1．onedriveの最新版をダウンロードし、インストール こちらのリンクからダウンロードし、インストールしてください。
2．再起動する そのままの意味です。普通に再起動してください。
3．onedriveの設定をする    このowndriveロゴを右クリックで開き、設定を選択します。  

設定画面が出てきたら、ファイルのオンデマンドというのが出てきているので、ここのチェックボックスにチェックを入れることで有効になります。
もし、ここでファイルのオンデマンドという項目が出てきていない場合は、何回か手順1を繰り返すとできる可能性があります。
それでも、うまくいない場合は下のコメント欄に質問していただけると、お役に立てるかわかりませんが、可能な限り返事させていただきますのでよろしくお願いいたします。
まとめ Windows10 Fall Creators Updateは大型アップデートの中では一番実用度が高いアップデートだと個人的には思っています。 今回の大型アップデートの醍醐味のひとつなのにこの問題はどうなのかなと思ってしまうところですが、windowsupdateでまだ供給されていないので仕方がないのかなと思う気持ちもあります。これが使えると容量不足に悩まされることなく、使えるので皆さんも使ってみてください。</description>
    </item>
    
  </channel>
</rss>